pkg/webhook/webhook.go

```diff
@@ -180,6 +181,10 @@ func (w *Config) createFleetWebhookConfiguration(ctx context.Context) error {
 		},
 	}
 
+	if err := bindWebhookConfigToFleetSystem(ctx, w.mgr.GetClient(), &whCfg); err != nil {
```

Comment:
why not just put this in the whCfg object meta?

Reply:
please also add a comment on why we add this owner reference

Reply:
> why not just put this in the whCfg object meta?

That is what the `bindWebhookConfigToFleetSystem` func is doing. I need to first make a call to get the NS resource so I have the UID. Which is why this call occurs after the construction of the whCfg object.

> please also add a comment on why we add this owner reference
Done